cumulus/primitives/storage-weight-reclaim/src/lib.rs

```diff
@@ -140,11 +140,11 @@ where
 	fn pre_dispatch(
 		self,
 		_who: &Self::AccountId,
-		_call: &Self::Call,
+		call: &Self::Call,
 		_info: &sp_runtime::traits::DispatchInfoOf<Self::Call>,
 		_len: usize,
 	) -> Result<Self::Pre, sp_runtime::transaction_validity::TransactionValidityError> {
-		Ok(get_proof_size())
+		Ok((get_proof_size(), call.clone()))
```

Comment:
Cloning the entire call is a little bit too expensive.

Reply:
What about adding the `GetCallMetadata` to `T::RuntimeCall` and debugging the corresponding `CallMetadata`?

Reply:
You can just see the transaction with all its arguments in the block explorer. I don't see any need to log them.

Reply:
Alright, I just pushed the changes as per your suggestion.